pitop/camera/camera_calibration/camera_calibration.py

```diff
@@ -0,0 +1,123 @@
+import numpy as np
```

Comment:
is this something that is generically useful for multiple applications? i am wondering if we can have this handled by the CLI - `pi-top camera calibrate` type thing?

Reply:
That's a good point actually yeah, let's do that!

Reply:
Was just thinking that this is a standalone application, same as the IMU calibration!

Reply:
https://github.com/pi-top/pi-top-Python-SDK/issues/267